ci.sh

```diff
@@ -153,10 +153,10 @@ activate_d() {
   download_install_sh "$install_sh"
   # DUB isn't needed for gdc
   if [ "${DMD:-dmd}" == "gdc" ] ; then
-      touch "$HOME/dlang/dub"
+      mkdir -p $HOME/dlang/dub
```

Comment:
Are we guaranteed that `$HOME` does not contain spaces?

Reply:
As this is only supposed to run on CI platforms, I'm fine with that assumption.